src/components/form/EventForm/Theme/EventThemeForm.tsx

```diff
@@ -116,9 +120,9 @@ export const EventThemeForm: React.FC<EventThemeFormProps> = ({event, onEventCha
                 </StudioSection>
 
                 <div className={formStyles.submit}>
-                    <PrimaryButton type={"submit"}>
+                    <Button type={"submit"}>
```

Comment:
no need for `{curly}` bois

Reply:
I know but I want to keep them for consistency 😄